Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/SignalsAndSlots.rst

```diff
@@ -41,44 +41,68 @@ like in this example:
 
 *Example: Emitting a Signal* ::
 
-	/**
-	 * @param Comment $newComment
-	 * @return void
-	 */
-	public function createAction(Comment $newComment) {
-		...
-		$this->emitCommentCreated($newComment);
-		...
-	}
+    /**
+     * @param Comment $newComment
+     * @return void
+     */
+    public function createAction(Comment $newComment) {
+        ...
+        $this->emitCommentCreated($newComment);
+        ...
+    }
 
 The signal will be dispatched to all slots listening to it.
 
 Defining Slots
 ==============
 
-Basically any method of any class can be used as a slot, even if never written
-specifically for being a slot. The only requirement is a matching signature between signal
-and slot, so that the parameters passed to the signal can be handed over to the slot
-without problems. The following shows a slot, as you can see it differs in no way from any
-non-slot method.
+You can use any method as a slot, or write one specifically for use s a slot.
+
+Any method of any class can be used as a slot, even if never written specifically
+for being a slot. The only requirement is a matching signature between signal and
```

Comment:
This sentence somehow repeats the previous sentence, just with slightly different words, no?

Reply:
Yes, but… I added two more headlines now.